### DISCUSSION TITLE
Using xDrip and Dexcom receiver at the same time: how to start new sensor.

### DISCUSSION THREAD

**Discussion by Agio56** [NightscoutFoundation/xDrip#2949] [Q&A]
I'm sorry if I ask a stupid question...
I use both the Dexcom receiver and xDrip with a One system. This morning the sensor expired and I had to start a new one, but I did something wrong and I'm really confused.
What are the steps to follow for a correct procedure? Stop the old sensor and start the new one with xdrip and the receiver simultaneously?

**Comment by Navid200** [comment]
If you must use the receiver, do the stopping and starting of the sensor from the receiver only.
xDrip, being connected to the transmitter, will know what's going on.

There is no need to also start the sensor on xDrip.  xDrip will know because it is  connected to the same transmitter.
There is no need to enter the calibration code in xDrip.  The only thing xDrip does with the calibration code is to verify that it is an acceptable code and if it is, submits it to the transmitter.  So, if you start from the receiver and enter the correct calibration code, that's all there is to it about the calibration code.
A consequence of this is that the calibration code shown on the xDrip system status page may be wrong.  You can just ignore that.

**Reply by Agio56** [reply]
Really very clear, thanks Navid.
I have another question for you :). 
In 10 days the transmitter currently in use will expire. How should I proceed to make xDrip recognize the new one?

Will it be enough to enter the new code in settings -> "Settings −> Dexcom Transmitter ID"?

**Reply by Navid200** [reply]
Before I answer that, I would like to clarify the last time you can start a sensor with a transmitter is when "Transmitter Days" shown on status page is 99.
This means, if transmitter days is showing 95 and you start a sensor and you let the sensor go for 10 days, you will not be able to start another sensor because at that point, transmitter days will be showing 105.
If you start a sensor when transmitter days is showing 99, it will go fine for 10 days.
If you attempt to start a sensor when transmitter days is showing 100, it will not start.

A new transmitter first needs to be activated.  That is done when you snap in the new transmitter into an inserted sensor.
If you want to activate a new transmitter before inserting a new sensor, you can do that by holding the new transmitter in your hand with your thump pressed against the two contacts for a while (1-2 minutes).

You should never activate a new transmitter unless when you intend to start using it.  After you activate a transmitter, it will start counting days and you will not be able to stop it.

If you have an activated new transmitter, all you need to do is to enter the transmitter ID in xDrip and be patient.
Unfortunately, xDrip will keep asking you to start sensor.  Please don't!  Do NOT start a sensor on a new transmitter until the battery voltages are shown on the status page.  This could take up to 30 minutes.  It's a time you may consider a waste.  But, it is worth being patient to avoid any possible sensor start failures that could happen with a brand new transmitter.
